starknet_devnet/dump.py

```diff
@@ -0,0 +1,43 @@
+"""Dumping utilities."""
+
+import multiprocessing
+
+import dill as pickle
+
+from .util import DumpOn
+
+def perform_dump(dumpable, path):
```

Comment:
Why is this not a class method?

Reply:
Not sure tbh, could be both, what do you think?

Reply:
If the dumper is the only one who'll perform the dumping we can move it to a class method

Reply:
Ended up renaming it and making it a private method: https://github.com/Shard-Labs/starknet-devnet/pull/40/commits/a9cfec85cf925ee5860c1b41918a81fdf783be76